src/app/core/metadata/metadata-value.model.ts

```diff
@@ -0,0 +1,94 @@
+import {link, typedObject} from '../cache/builders/build-decorators';
```

Comment:
this class (or similar) was not there before?

Reply:
There are similar metadataValue classes but no one is capable for our issue.